cirq-google/cirq_google/api/v2/program.proto

```diff
@@ -106,17 +112,111 @@ message Language {
   string arg_function_language = 2;
 }
 
+// Argument that is constrained to a float or symbolic expression
+message FloatArg {
+  oneof arg {
+    float float_value = 1;
+    string symbol = 2;
+    ArgFunction func = 3;
+  }
+}
+
+// Representation of cirq.XPowGate
+message XPowGate {
+  FloatArg exponent=1;
+}
+
+// Representation of cirq.YPowGate
+message YPowGate {
+  FloatArg exponent=1;
```

Comment:
nit: add whitespace around `=` for consistency, in several places in this file.

Reply:
Done.